src/views/Scenario/Scenario.js

```diff
@@ -65,9 +65,12 @@ const Scenario = (props) => {
   const workspaceId = workspace.data.id;
   const [editMode, setEditMode] = useState(false);
 
-  function handleScenarioChange (event, scenario) {
+  const currentScenarioRenderInputToolType = editMode ? 'views.scenario.dropdown.scenario.tooltip.disabled' : '';
```

Comment:
Could you please extract 'views.scenario.dropdown.scenario.tooltip.disabled' into a Constant file?

Reply:
Where it is done elsewhere in the code ???

Reply:
Wont do. It must be done in one separated ticket